contents/blog/real-user-monitoring.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Real user monitoring: what it is, why do it, and how to set it up"
+date: 2023-05-31
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/happy-hog.png
+featuredImageType: full
+category: Product growth
+tags:
+  - Guides
+  - Explainers
+---
+
+Before anyone can use your product, you must make sure it actually works. Figuring this out is more tricky than just running it locally. Changes often have unintended consequences on other parts of the app or create non-obvious issues. To track and prevent this, companies rely on monitoring.
+
+There are a couple of ways to do monitoring. The first is synthetic monitoring which relies on  simulated behaviors and software tests. The second way, which we cover in this post, is real user monitoring. This is relying on the behavior of real users to ensure your product is working as intended. 
+
+## Real user monitoring in detail
+
+Real user monitoring is tracking the usage, performance, and quality of your product based on how users are experiencing it. It is different from synthetic monitoring which tracks similar metrics but with tests and simulations. 
+
+Because of this difference, real user monitoring is a closer representation of reality. It identifies real issues, largely broken down into two classes:
+
+- Performance: loading, page, and query speed
+- Usability: errors, confusion, bugs
+
+![Types](../images/blog/real-user-monitoring/classes.png)
+
+The ideal real user monitoring system identifies these problems, figures out their details, supports improvement prioritization, and ultimately, creates a better overall user experience.
```

Comment:
> supports improvement prioritization

What is improvement prioritization?

Reply:
reworded

Reply:
I dont think the change has been uploaded?